kuber/kube.go

```diff
@@ -951,3 +952,20 @@ func (kube *Kube) GetAgentPermissions() (string, error) {
 	rules, _ := json.Marshal(subjectRules.Status.ResourceRules)
 	return string(rules), nil
 }
+
+func (kube *Kube) GetMinorVersion() (int, error) {
```

Comment:
When you pull from dev and master, there's probably gonna be duplication with the function I added to get the version. Can you please handle it?